spade-core/src/main/scala/com/salesforce/mce/spade/cli/CliOptions.scala

```diff
@@ -28,7 +29,8 @@ object CliOptions {
     workflowId = "",
     pipelineName = "",
     activate = false,
-    compact = false
+    compact = false,
+    array = false
```

Comment:
I think this should always be true, right? single workflow is one corner case with size = 1.

Reply:
@anniezhang77 going forward `--array` should always be true, for the interim, this is acting as the new vs old flag so the non `--array` ones still follow the old serializing method, but we do need to redeploy all deployed pipelines to update to add `--array` before we redeploy Sprinkler with the new handling code

Reply:
got it. What I meant is we probably don't need this flag, and workflows are always encoded/decoded as array.

Reply:
@anniezhang77 that is also a valid option where the new vs old is shifted to pipeline spade-ext version bump PR -> CI deployment, @realstraw any preference here?